### ISSUE TITLE
Next doesn't launch (OS X, sierra)

### ISSUE THREAD

**Issue by peterohanley** [atlas-engineer/next#158]
Next bounces a few times and then closes. Using a terminal to get an error message:

    $ open -a Next -n
    LSOpenURLsWithRole() failed for the application /Applications/Next.app with error -10810.

I tried deleting the `com.apple.quarantine` xattrs but it stops opening at all from spotlight, and using a terminal gives the same result.

**Comment by Ambrevar** [comment]
But it builds from source on older macOS, right?

**Comment by jmercouris** [comment]
Pierre Neidhardt <notifications@github.com> writes:

Yes, it should be able to build from source on an older version of
macOS

> But it builds from source on older macOS, right?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub, or mute the thread.
>
<#secure method=pgpmime mode=encrypt>

**Comment by totallymike** [comment]
I'm getting this error on 10.14.2 as well

**Comment by Colelyman** [comment]
I'm getting this error on 10.14.1 (18B75) too

**Comment by jorgeavaldez** [comment]
I'm having this issue on 10.14.1 (18B75) using the prebuilt binary.

**Comment by bjb568** [comment]
Same here on 10.14.2 (18C52a). Trying to install from source, `make cocoa-webkit` gives `fatal error: 'xmlrpc-c/base.h' file not found`.

**Comment by jorgeavaldez** [comment]
Looks like there might be a possible fix coming with https://github.com/atlas-engineer/next/pull/168 since it addresses your issue @bjb568

**Comment by jmercouris** [comment]
yeah, I am seriously thinking about bundling XML-RPC-C within the Next repository...

**Comment by jmercouris** [comment]
@bjb568 could you please try pulling the source again and compiling, it should work now

**Comment by jmercouris** [comment]
I should probably also update the binary as well, since the info plist has changed. I've uploaded a new version of the binary with all the changes: https://next.atlas.engineer/download please give it a try and let me know I this fixes your issues.

**Comment by Colelyman** [comment]
@jmercouris fixes the issues for me! (On 10.14.1 (18B75)) Thanks!

**Comment by bjb568** [comment]
@jmercouris I'm getting this error.
```
.../next-master/ports/cocoa-webkit/ArgParse/argparse.c:84:9: error: use of undeclared identifier 'errno'
        errno = 0; 
        ^
.../next-master/ports/cocoa-webkit/ArgParse/argparse.c:94:13: error: use of undeclared identifier 'errno'
        if (errno) 
            ^
.../next-master/ports/cocoa-webkit/ArgParse/argparse.c:95:48: error: use of undeclared identifier 'errno'
            argparse_error(self, opt, strerror(errno), flags);
                                               ^
.../next-master/ports/cocoa-webkit/ArgParse/argparse.c:100:9: error: use of undeclared identifier 'errno'
        errno = 0; 
        ^
.../next-master/ports/cocoa-webkit/ArgParse/argparse.c:110:13: error: use of undeclared identifier 'errno'
        if (errno) 
            ^
.../next-master/ports/cocoa-webkit/ArgParse/argparse.c:111:48: error: use of undeclared identifier 'errno'
            argparse_error(self, opt, strerror(errno), flags);
                                               ^
6 errors generated.
```

**Comment by Ambrevar** [comment]
@jmercouris: You might an `#include <errno.h>` or similar.
